dockerfile-generator/dockerfile-gen-services/src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.main.banner-mode=off
```

Comment:
make it a controllable through an env to on and off .
we need this for automation to parse outputs

Reply:
This property is only to disable the banner at module level i.e. dockerfile gen that too during spring boot test. If enabled it prints the Spring boot banner. Do we have a case where we might need that banner?

Reply:
With the latest changes it was made a controllable prop .

Reply:
This is a test level property to skip printing banner when spring boot initializes the module. Being at the module level it won't go through the controllable props.